### ISSUE TITLE
settings menu is blank

### ISSUE THREAD

**Issue by theotheroracle** [hbang/NewTerm#71]
i have an iPhone 6, running ios 12.5.2, jailbroken with checkra1n and odysseyra1n . ( sileo / proscurus jailbreak )

when i open the settings menu, it' just blank .

interesting note, i have another iPhone 6 running ios 12.5.4, jailbroken with unc0ver, ( cydia / substitute jailbreak ) and the settings menu works on that phone .

**Comment by theotheroracle** [comment]
ok after installing libhooker, the problem seems to be resolved, so maybe that should be added as a dependency ?

**Comment by kirb** [comment]
Ah… I know why. While I did remove Substrate/libhooker from the dependencies of Cephei in the 1.17 release last week, CepheiPrefs still relies on it. I’ll need to change CepheiPrefs to use Logos’ internal generator so the Substrate dependency isn’t needed.

This should only happen if you haven’t already installed tweaks prior to installing NewTerm, of course.

**Comment by theotheroracle** [comment]
yeah newterm was the first thing i installed lol, it's the most important software on my devce ;p
